db/migrate/20220118154227_create_sick_leaves.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSickLeaves < ActiveRecord::Migration[6.1]
+  def change
+    create_table :sick_leaves do |t|
+      t.string :destination
+      t.integer :doctor_id, null: false
+      t.integer :patient_id, null: false
+      t.datetime :start_at, null: false
+      t.datetime :end_at, null: false
```

Comment:
I would rather rename these fields
You can use `start_date` and `end_date` or `started_at` and `ended_at`
Otherwise it's not grammatically correct